config/registry_image_pruner/image_pruner/prune_images.py

```diff
@@ -91,8 +91,11 @@ def remove_tags(tags: List[Dict[str, Any]], quay_token: str, namespace: str, nam
                 else:
                     LOGGER.info("Removing tag %s from %s/%s", tag["name"], namespace, name)
                     delete_image_tag(quay_token, namespace, name, tag["name"])
+        elif tag["name"].endswith(".src"):
+            LOGGER.info("Removing deprecated tag %s", tag["name"])
+            delete_image_tag(quay_token, namespace, name, tag["name"])
```

Comment:
Discussed in Slack: we should only remove these if the corresponding binary image doesn't exist

Otherwise, we could delete the only source container tag (i.e. completely delete the source container) for an image (for old images, built before we started pushing the new tag).

Reply:
Updated. PTAL.